### ISSUE TITLE
Web Extension

### ISSUE THREAD

**Issue by Brettm12345** [Brettm12345/github-moonlight#29]
It would be cool to build this as a web extension with the ability to easily switch colors, add custom themes and change settings

**Comment by gregk-git** [comment]
Would also increase the number of users that will find it. Should that get a different git repo, considering that will function differently, and probably require JS?

**Comment by Optimalgitit** [comment]
I would say TS is a good choice. Strong Typing so we don't miss an error.

**Comment by Brettm12345** [comment]
Yeah I could build the UI in React. It would be pretty simple

**Comment by Brettm12345** [comment]
There's definitely more work to do before we get to this point but I thought it would be a better solution

**Comment by Brettm12345** [comment]
> I would say TS is a good choice. Strong Typing so we don't miss an error.

The build script is written in TypeScript [build.ts](../blob/master/build.ts)

**Comment by Optimalgitit** [comment]
Yes, I recognize it's in TS. The CSS would actually be really easy to port, as you can directly modify the webpage in WebExtensions. I don't know about react, however. It's just going to be an extension.
